### ISSUE TITLE
[Chip] Chip's Icon not showing

### ISSUE THREAD

**Issue by eriklimakc** [unoplatform/uno.toolkit.ui#557]
## Current behavior

Chip's Icon not being displayed

![image](https://user-images.githubusercontent.com/114886335/234912001-d92dd50f-e13d-4d35-8eef-51e46aa501a3.png)

## Expected behavior
The icon to be displayed

## How to reproduce it (as minimally and precisely as possible)

First I tried with the Figma Plugin, then with an Uno App and Playground (Here's the link to the snippet -> http://playground.platform.uno/#7c595c83)

Basically, add a ChipGroup and a Chip with Icon.

Uno.UI v4.8.15
Uno.Themes v2.5.3
Uno.Toolkit.UI v2.5.8
Uno.Extensions v2.3.2

Affected platform(s):

All

IDE:

- [X] Visual Studio 2022
- [ ] Visual Studio 2019
- [ ] Visual Studio Code
- [ ] Visual Studio for Mac
- [ ] Rider Windows
- [ ] Rider macOS

Relevant plugins:

**Comment by kazo0** [comment]
instead of PathIcon can you try just using an Image element? I dont think the Icon property supports types of IconElement, it probably should though

although you could probably use IconTemplate and use a PathIcon inside of the DataTemplate

**Comment by Xiaoy312** [comment]
oculd you produce a [TreeGraph](https://github.com/unoplatform/uno/blob/cd501cf79cbc8ee62cbafbd325c870d8be42760e/src/Uno.UI/Extensions/ViewExtensions.visual-tree.cs#L44) of the rendered output?

**Comment by eriklimakc** [comment]
@Xiaoy312 Not sure of how to produce a [TreeGraph](https://github.com/unoplatform/uno/blob/cd501cf79cbc8ee62cbafbd325c870d8be42760e/src/Uno.UI/Extensions/ViewExtensions.visual-tree.cs#L44). Should I call it on my Page?

By looking at the Toolkit Sample of Chips, I realized that Chip Icon only renders if the Chip is NOT a child of a `ChipGroup`. For example:

```
<utu:Chip Content="Chip Icon Image" Style="{StaticResource InputChipStyle}">
  <utu:Chip.Icon>
        <Image Source="ms-appx:///MyApp/Assets/Avatar.png" />
  </utu:Chip.Icon>
</utu:Chip>
<utu:Chip Content="Chip Icon PathIcon" Style="{StaticResource InputChipStyle}">
  <utu:Chip.Icon>
	  <PathIcon Data="{StaticResource Icon_Delete}" />
  </utu:Chip.Icon>
</utu:Chip>
```

But if I add a parent `ChipGroup` to those Chips their icons won't render anymore, even if it's a `<Image />`

**Comment by Xiaoy312** [comment]
yea, you will need to add a button to your MainPage (in BottomPanel area)
with the handler: 
```
var tree = Uno.UI.Extensions.ViewExtensions.TreeGraph(renderOutput);
Console.WriteLine(tree);
```
